### PR TITLE
Fixed #209

### LINKED ISSUE #209: Youtube script doesn't support .be domain
Script can't handle URLs from youtu.be, only youtube.com.

### PR DESCRIPTION
**Pull request by drobati**
This regex should fix the problem.

### CODE REVIEW

**Comment by sshirokov** [comment]
This fucks up the match groups if the second clause is used:

this

vs 

that

Note the match index. The `youtu.be` case will fail to find the URL from where I'm sitting.

**Comment by drobati** [comment]
@sshirokov are you sure that the latter is a valid case? I think youtu.be domain doens't use watch?=v.

**Comment by drobati** [comment]
https://regex101.com/r/wU6cO9/1

**Comment by drobati** [comment]
note: https://regex101.com/r/wU6cO9/2

**Comment by drobati** [comment]
Should work now but I have no idea if it works since I can't get vagrant to work and there are no tests.

**Comment by sshirokov** [comment]
What you did here is now break the first case, not fix anything... :crying_cat_face:

File: scripts/youtube-info.coffee [hunk 1]
```diff
@@ -35,4 +35,5 @@ getTitle = (msg, url) ->
     else
       $ = cheerio.load(body)
       title = $('title').text()
+      console.log title
```
Comment (sshirokov) [inline scripts/youtube-info.coffee]: Can you nuke this also, as well.

**Comment by drobati** [comment]
@sshirokov Done.

**Comment by drobati** [comment]
Fuck left a done callback.

**Comment by sshirokov** [comment]
who the fuck even needs tests

:fu: tests

**Pull request merged**